Lib/typing.py

```diff
@@ -1775,6 +1777,12 @@ def _strip_annotations(t):
         if stripped_args == t.__args__:
             return t
         return GenericAlias(t.__origin__, stripped_args)
+    if isinstance(t, types.Union):
```

Comment:
Just double checking, this test case (and the other one in `_eval_type`) is covered by the `get_type_hints` test cases above right?

Reply:
I have added test to cover this case